### PR TITLE
Added an email counter

### LINKED ISSUE #870: Add a counter for EmailInput component
A counter that will change on each valid email entered/removed from the EmailInput component.

cc: @amaldinesh7

### PR DESCRIPTION
**Pull request by Thejus-Paul**
Fixes #870 
- Added an email counter and used it on Formik story.

### CODE REVIEW

**Comment by Thejus-Paul** [comment]
email_counter

@amaldinesh7 _a Please review.

**Comment by subins2000** [comment]
@Thejus-Paul Hi Thejus, just a suggestion: Notice how the input field's height changes when emails are added. I think it's better if the height of input field remains the same whether or not there are any items in it. I came across this in neetoChat where in a form the email input is bigger than every other field in the form.

**Comment by Thejus-Paul** [comment]
@subins2000 I also found that to be an issue. Please check the image below. After increasing the size of the email input field, it didn't feel good to me. This is after making the select component below it to size "large". 
image
The image below is how it is rendering currently and this one felt better to me.
image
When the user types, maybe because of the motion or because of the content inside it, I didn't feel the increase in the input field that much awkward to me.
image
What is your thought on this @amaldinesh7 ?

File: lib/constants/EmailInput.js [hunk 1]
```diff
@@ -32,3 +32,10 @@ export const pruneDuplicates = inputValues => {
   const uniqueValues = [...new Set(values)];
   return uniqueValues.map(email => formatEmailInputOptions(email));
 };
+
+export const renderValidEmails = values =>
+  values.filter(email => email.valid && email);
+
+export const getEmailsCount = values => renderValidEmails(values).length;
```
Comment (amaldinesh7) [inline lib/constants/EmailInput.js:39]: These ain't constants. Please move all the function to helpers/utils

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]
@Thejus-Paul _a Please check the comment

**Comment by amaldinesh7** [comment]
> @subins2000 I also found that to be an issue. Please check the image below. After increasing the size of the email input field, it didn't feel good to me. This was after making the select component below it to a size large. image The image below shows how it is rendering currently. This one felt better to me. image When the user types, maybe because of the motion or the content inside it, I didn't feel the increase in the input field that much awkward to me. image What is your thought on this @amaldinesh7 ?

Let's not make this change. Increased height in the default state looks weird. You can adjust the height of the container in which you use this component, such that it won't make a jerk to the UI. 

cc: @praveen-murali-ind

**Comment by Thejus-Paul** [comment]
@amaldinesh7 _a Please review.

**Comment by Thejus-Paul** [comment]
@amaldinesh7 _a Please review.

**Review by amaldinesh7 — APPROVED** [review]

**Pull request merged**